subprojects/gradle/android/src/main/kotlin/com/avito/android/ProjectExtensions.kt

```diff
@@ -26,6 +29,10 @@ fun Project.withAndroidModule(block: (testedExtension: TestedExtension) -> Unit)
     withAndroidLib(block)
 }
 
+@Suppress("UnstableApiUsage")
```

Comment:
Also, where are loads of TODOs in sources
I propose to use this new API only when we don't have an alternative.